packages/daemon/src/runner/job-runner.ts

```diff
@@ -84,6 +84,15 @@ export class JobRunner {
     }
   }
 
+  /**
+   * extract plugin specific param from job param
+   * @param type plugin type
+   * @param params job param
+   */
+  private getJobParam(type: PluginTypeI, params: JobParam[]) {
+    return params ? params.filter((it) => it.pluginType === type).map((it) => it.data) : [{}];
```

Comment:
```suggestion
    return params ? params.filter((it) => it.pluginType === type).map((it) => it.data) : [];
```

Does this just return an empty array if params not exist?

Reply:
Oops, why does this API return an array instead of an item(`r[0].data`)?